src/Illuminate/Broadcasting/Broadcasters/ReboundBroadcaster.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Illuminate\Broadcasting\Broadcasters;
+
+use Illuminate\Contracts\Broadcasting\Broadcaster;
+use Illuminate\Contracts\Redis\Database as RedisDatabase;
+
+class ReboundBroadcaster extends AbstractBroadcaster implements Broadcaster
+{
+    /**
+     * The Redis instance.
+     *
+     * @var \Illuminate\Contracts\Redis\Database
+     */
+    protected $redis;
+
+    /**
+     * The Redis connection to use for broadcasting.
+     *
+     * @var string
+     */
+    protected $connection;
+
+    /**
+     * Create a new broadcaster instance.
+     *
+     * @param  \Illuminate\Contracts\Redis\Database  $redis
+     * @param  string  $connection
+     * @return void
+     */
+    public function __construct(RedisDatabase $redis, $connection = null)
+    {
+        $this->redis = $redis;
+        $this->connection = $connection;
+    }
+
+    /**
+     * Return the valid Rebound authentication response.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  mixed  $result
+     * @return mixed
+     */
+    public function validAuthenticationResponse($request, $result)
+    {
+        return ['status' => 'success', 'user_id' => $request->user()->id, 'user_info' => $request->user()];
```

Comment:
this is a custom broadcast driver, not the core redis driver. agreed what your saying is true, but this is a custom build package, the default redis broadcaster returns false. i will update this to reflect the same api.